modules/admin-ui-frontend/app/scripts/shared/partials/modals/series-details.html

```diff
@@ -388,41 +388,43 @@ <h2 translate="EVENTS.SERIES.DETAILS.HEADER" translate-values="{{titleParams}}">
       <div class="full-col">
         <div class="obj list-obj">
           <header translate="EVENTS.SERIES.DETAILS.TABS.TOBIRA"></header>
-          <div class="obj-container">
-            <a translate="EVENTS.SERIES.DETAILS.TOBIRA.DIRECT_LINK" href="{{directTobiraLink}}"></a> <a ng-click="copyTobiraDirectLink()" aria-label="{{'EVENTS.SERIES.DETAILS.TOBIRA.COPY_DIRECT_LINK' | translate}}">
-              <i aria-hidden="true" class="fa fa-copy" title="{{'EVENTS.SERIES.DETAILS.TOBIRA.COPY_DIRECT_LINK' | translate}}"></i>
-            </a>
```

Comment:
One could argue that the direct link could/should always be shown? Because the link will always stay the same and we don't need any Tobira data to construct it.

Reply:
Yeah, but that wasn't the case before: The link was shown, but it was wrong in most real life situations. This strictly improves the situation. Getting it right would require effort outside the scope of this PR IMO.